it/server/src/test/java/com/linecorp/centraldogma/it/TestAllReplicasPlugin.java

```diff
@@ -43,8 +45,8 @@ public CompletionStage<Void> stop(PluginContext context) {
     }
 
     @Override
-    public Class<?> configType() {
+    public Class<? extends PluginConfig> configType() {
         // Return the plugin class itself because it does not have a configuration.
-        return TestAllReplicasPlugin.class;
+        return NoopPluginConfig.class;
```

Comment:
Discussion point 3.

From some reason. Plugins have specification for duplication check in plugin group.

It seems weird to check duplication by plugin config class because for that check, some plugins which has no configurations returns itself.

Plugin config may be resuable, then duplication check key should be plugin class, not plugin config.

HDYT?